src/main/java/com/jcabi/dynamo/mock/H2Data.java

```diff
@@ -259,6 +259,20 @@ public void update(final String table, final Attributes keys,
     }
 
     /**
+     * Delete attributes from the given table.
+     * @todo 30min:#35 Implement H2Data#delete() method because MkData#delete
```

Comment:
@Suseika the puzzle syntax is wrong here. first the issue number and then the time `@todo #35:30min ...`